### PR TITLE
Add log info for debugging

### PR DESCRIPTION
**Pull request by zyiou**
Log the error values in Encode function to assist debugging.

### CODE REVIEW

File: pkg/intermediate/aggregate.go [hunk 1]
```diff
@@ -237,15 +237,15 @@ func (a *AggregationProcess) correlateRecords(incomingRecord, existingRecord ent
 				if ieWithValue.Value != "" {
 					existingIeWithValue, _ := existingRecord.GetInfoElementWithValue(field)
 					if existingIeWithValue.Value != "" {
-						klog.Warningf("This field with name %v should not have been filled with value %v in existing record.", field, existingIeWithValue.Value)
+						klog.Warningf("This field with name %v and value %v should not have been filled with value %v in existing record.", field, ieWithValue.Value, existingIeWithValue.Value)
```
Comment (srikartati) [inline pkg/intermediate/aggregate.go:240]: This log is not clear. If we need current value, lets do this:
" %v field should not have been filled in the existing record; existing value: %v and current value: %v"

**Review by srikartati — COMMENTED** [review]
A comment on the clarity of logs.

File: pkg/intermediate/aggregate.go (hunk 1 above)
Comment (zyiou) [reply pkg/intermediate/aggregate.go]: Done. Thanks!

**Review by srikartati — APPROVED** [review]
LGTM.

**Pull request merged**